challenges/interactive_pricing_component/index.html

```diff
@@ -26,38 +26,212 @@
         >
-        Internship.
+          <div
+            class="text-dark-desaturated-blue flex flex-col items-center lg:items-start"
+          >
+            <div class="flex items-center py-2">
+              <img
+                src="images/icon-check.svg"
+                alt="check icon"
+                class="h-full"
+              />
+              <p class="px-2">Unlimited websites</p>
+            </div>
+            <div class="flex items-center py-2">
+              <img
+                src="images/icon-check.svg"
+                alt="check icon"
+                class="h-full"
+              />
+              <p class="px-2">100% data ownership</p>
+            </div>
+            <div class="flex items-center py-2">
+              <img
+                src="images/icon-check.svg"
+                alt="check icon"
+                class="h-full"
+              />
+              <p class="px-2">Email reports</p>
+            </div>
+          </div>
+          <div class="pt-4">
+            <button
+              class="bg-dark-desaturated-blue rounded-3xl px-8 py-2 text-white"
+            >
+              Start my trial
+            </button>
+          </div>
+        </div>
       </div>
+      <div
+        class="relative bottom-20 flex w-full flex-col place-content-center text-center"
+      >
+        <div>
+          <a
+            class="text-optimum-blue hover:text-optimum-darkblue"
+            href="https://www.frontendmentor.io/challenges/interactive-pricing-component-t0m8PIyY8"
+            target="_blank"
+            >Frontend Mentor challenge</a
+          >
+        </div>
+        <div>
+          for
+          <a
+            class="text-optimum-blue hover:text-optimum-darkblue"
+            href="https://github.com/optimumBA"
+            target="_blank"
+            >Optimum BH</a
+          >
+          Internship.
+        </div>
 
-      <div class="mt-3">
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://github.com/optimumBA/frontend_mentor_challenges/tree/interactive_pricing_component/challenges/interactive_pricing_component"
-          >Source code</a
-        >
+        <div class="mt-3">
+          <a
+            class="text-optimum-blue hover:text-optimum-darkblue"
+            href="https://github.com/ngumokenneth/frontend_mentor_challenges/tree/interactive_pricing_component/challenges/interactive_pricing_component"
+            >Source code</a
+          >
+        </div>
       </div>
     </div>
+
+    <script>
+      document.addEventListener('alpine:init', () => {
+        Alpine.data('app', () => ({
+          data: '3',
+          yearly: false,
+          pageviews: ['10K', '50K', '100K', '500K', '1M'],
+          price: ['$8.00', '$12.00', '$16.00', '$24.00', '$36.00'],
+          yearlyPrice: ['$6.00', '$9.00', '$12.00', '$18.00', '$27.00'],
+
+          getIndex(val) {
+            if (val == '1') {
```

Comment:
Please refactor this.
What would this code look like if there were 20 questions?